src/message-dispatch.js

```diff
@@ -25,6 +25,13 @@ export default class MessageDispatch extends EventTarget {
   addToPresenceLog(entry) {
     entry.key = Date.now().toString();
 
+    const lastEntry = this.presenceLogEntries.length > 0 && this.presenceLogEntries[this.presenceLogEntries.length - 1];
+    if (lastEntry && entry.type === "permission" && lastEntry.type === "permission") {
+      if (lastEntry.body.permission === entry.body.permission && entry.key - lastEntry.key < 10000) {
```

Comment:
`key` is technically a string so its slightly weird to be subtracting them, but loljavascript it converts them an ddoes the "right" thing anyway

Reply:
Yeah, we are relying on the key for the timestamp as it should be unique, that's why this.

Reply:
Oh yep, I've updated to parse.